packages/table/src/common/internal/scrollUtils.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { IRegion, RegionCardinality, Regions } from "../../regions";
+
+// Public
+// ======
```

Comment:
not sure these headings are useful... the keyword `export` gets the point across.

Reply:
Removed